Library/Formula/libodb-boost.rb

```diff
@@ -0,0 +1,17 @@
+class LibodbBoost < Formula
+  homepage "http://www.codesynthesis.com/products.odb"
+  url "http://codesynthesis.com/download/odb/2.3/libodb-boost-2.3.0.tar.gz"
+  sha1 "38c8552db872cc32144fb653c19277cfec1df2ec"
+
+  depends_on "libodb"
+  depends_on "boost"
+  
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!